src/core/effects/qgsimageoperation.cpp

```diff
@@ -691,9 +657,7 @@ void QgsImageOperation::GaussianBlurOperation::operator()( QgsImageOperation::Im
       sourceRef = sourceFirstLine;
       destRef = reinterpret_cast< QRgb * >( outputLineRef );
       for ( int x = 0; x < width; ++x, ++destRef, sourceRef += 4 )
-      {
-        *destRef = gaussianBlurVertical( y, sourceRef, sourceBpl, height );
-      }
+        * destRef = gaussianBlurVertical( y, sourceRef, sourceBpl, height );
```

Comment:
It's either the combination of `remove_brackets` with either `pad-oper` (operator padding) or `align-pointer=name`. I would suspect it's with operator but will have to check and maybe report an error in astyle.